src/store/index.ts

```diff
@@ -1,12 +1,39 @@
 import { createStore } from 'vuex'
+import axios from 'axios'
+import router from '../router'
+
+const auth_module = {
+  actions: {
+      LOGIN (context, credentials){
+          axios.post('http://127.0.0.1:8000/api/login', credentials.values).then((response) => {
```

Comment:
Add try and catch to handle errors. make sure that error is also visible on frontend.